src/components/common/scheduleTable.jsx

```diff
@@ -15,7 +15,7 @@ const ScheduleTable = ({ services, renderDayOfWeek }) => {
         {services.map(service => (
           <tr key={service._id}>
             <td align="center">
-              <Icon iconClass={service.type.toLowerCase()} />
+              <Icon iconClass={service.icon.toLowerCase()} />
```

Comment:
this should probably be destructured, so it's a bit safer.

Reply:
I also wonder if this should include an aria attribute since, its only for display.